packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -114,10 +112,58 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
               data-ui-id="meeting-composite-people-button"
               disabled={props.disableButtonsForLobbyPage}
               label={meetingStrings.peopleButtonLabel}
+              styles={commonButtonStyles}
+            />
+          )}
+          {isEnabled(meetingCallControlOptions?.chatButton) !== false && (
+            <ChatButton
+              checked={props.chatButtonChecked}
+              showLabel={true}
+              onClick={props.onChatButtonClicked}
+              data-ui-id="meeting-composite-chat-button"
+              disabled={props.disableButtonsForLobbyPage}
+              label={meetingStrings.chatButtonLabel}
+              styles={commonButtonStyles}
             />
           )}
-        </Stack.Item>
+        </Stack>
       )}
     </Stack>
   );
 };
+
+const buttonContainerStyle: IStyle = {
+  padding: '0.75rem',
+  columnGap: '0.5rem'
+};
+
+const controlBarStyles: BaseCustomStyles = {
+  root: buttonContainerStyle
+};
+
+const commonButtonStyles: ControlBarButtonStyles = {
```

Comment:
Thia already looks so much nicer! My concern with not exporting these as buttons is if contoso wants to inject a custom button - there's no concrete way for them to style the buttons the same as ours

Reply:
(also I'm losing faith our react-components are actually providing good value the more we choose not to export :( )

Reply:
I was under the impression that we were going to add more buttons to react-composites and then use them here :o

Reply:
@JamesBurnside we _could_ export this style constant from `react-composites` as `MeetingControlButtonDefaultStyle` that they use.

I'm not convinced that stylistic differences are large enough to warrant a new set of buttons / base button.

Reply:
I agree, I hate having to export new buttons just for styling differences, but I also really want someone to be able to inject a custom button that looks and behaves the exact same as ours --

Reply:
Scratch my concern -- the API for custom buttons is simply props not JSX so by default we will style their buttons to match ours (they can pass in more styles for more customization)
We can work with Sophy's team to give them the necessary styles for creating their Azure Portal components

Reply:
Oh yeah, I had also completely misunderstood my own approach. Any custom buttons injected in the `MeetingComposite` *will* by default get the new styling.

But I'm generally very wary of the (stability of) all this CSS stacking.